build-system/tasks/validator.js

```diff
@@ -53,6 +64,9 @@ validator.flags = {
   'update_tests': 'Updates validation test output files',
 };
 
+validator.description = 'Builds and tests the AMP C++ validator.'
```

Comment:
And this should be `validatorCpp.description`

Reply:
Done